infra/src/dsl/xroad.ts

```diff
@@ -603,3 +603,13 @@ export const HousingBenefitCalculator = new XroadConf({
       '/k8s/xroad/client/HOUSING_BENEFIT_CALCULATOR_PASSWORD',
   },
 })
+
+export const ShipRegistry = new XroadConf({
+  env: {
+    XROAD_SHIP_REGISTRY_PATH: {
+      dev: 'IS-DEV/GOV/10017/Samgongustofa-Protected/skipaskra-V1',
+      staging: 'IS-DEV/GOV/10017/Samgongustofa-Protected/skipaskra-V1',
```

Comment:
FYI this will probably throw an error in XRoad staging.

Reply:
Yep, I thinks it's strange that so many organizations aren't providing us with staging services on xroad but that is the case here